lib/cam.js

```diff
@@ -658,13 +658,16 @@ Cam.prototype.getActiveSources = function() {
 
 		this.activeSources[idx] = {
 			sourceToken: videoSource.$.token
-			, profileToken: this.defaultProfiles[idx].$.token
-			, encoding: this.defaultProfiles[idx].videoEncoderConfiguration.encoding
-			, width: this.defaultProfiles[idx].videoEncoderConfiguration.resolution.width
-			, height: this.defaultProfiles[idx].videoEncoderConfiguration.resolution.height
-			, fps: this.defaultProfiles[idx].videoEncoderConfiguration.rateControl.frameLimit
-			, bitrate: this.defaultProfiles[idx].videoEncoderConfiguration.rateControl.bitrateLimit
+			, profileToken: this.defaultProfiles[idx].$.token 
 		};
```

Comment:
![image](https://user-images.githubusercontent.com/30803642/84218147-69b46080-ab00-11ea-8b29-75c183c98388.png)
In some cases, a null pointer may appear here

Reply:
Thanks!